index.js

```diff
@@ -13,7 +13,7 @@ module.exports = pagination = async ({interaction, message, replyMessage = false
    if (buttonList.length < 2) throw new Error("Need provide at least 2 buttons");
    if (buttonList.length > 5) {
       process.emitWarning("More than 5 buttons have been provided the extras will be removed, remove the extra buttons from the buttonList to stop getting this message");
-      buttonList = buttonList.slice(0, 5);
+      buttonList = buttonList.slice(0, 4);
```

Comment:
Why this is changed 4 if the total max buttons was 5? It would return incomplete buttons

5 Buttons in readme
```js
// To use the first and last buttons use
buttonList = [
   button5, // First page button
   button1, // Next page button
   button2, // Previous page button
   button4, // Last page button
   button3 // Optional delete button (do not include if you do not want it)
]
```

concept:
```js
const list = ['Content1','Content2','Content3','Content4','Content5','unnecessaryContent6'];

console.log(list.splice(0, 5)); // Returns ['Content1','Content2','Content3','Content4', 'Content5']
console.log(list.splice(0, 4)); // Returns ['Content1','Content2','Content3','Content4']
```

Reply:
yeah that was an accident ill change it back thanks for point that out